hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -50,6 +50,48 @@ set_rule_status -rule {W_GLITCH} -status {Waived}          \
     (GlitchOutput=~"*u_sync_1*")} \
   -comment {Waive PADs input goes into synchronizer}
 
+# Driving from PAD RESET to SPI TPM 
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"IO*") && (ReceivingFlop=~"*u_spi_device.*")} \
```

Comment:
do we need to make this one more specific? this seems to say any IO pad to anything in spi_device...

Reply:
Right. I will place more specific paths instead of a common broad one.

Reply:
It's updated with u_spi_device.u_spi_tpm* which is the common path for following signals.
Screen Shot 2022-07-25 at 2.51.50 PM

Reply:
thanks!